e2e-polybft/framework/test-cluster.go

```diff
@@ -47,6 +47,9 @@ const (
 
 	// prefix for validator directory
 	defaultValidatorPrefix = "test-chain-"
+
+	// after how many blocks we consider block is finalized
+	defaultBlockFinalityDepth = 0
```

Comment:
Do we need this `constant`?

Reply:
This has been remove in latest commit https://github.com/0xPolygon/polygon-edge/pull/1228/commits/eecb93dddc6af118fab9d94be99140b748755a51